de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5555030_gh6376_FixEmptyTextInCustomerLabelName.sql

```diff
@@ -0,0 +1,41 @@
+drop VIEW "de.metas.fresh".product_specifications_v;
+CREATE OR REPLACE VIEW "de.metas.fresh".product_specifications_v AS
+  select
+    p.Name                                                  as productName,
+    p.CustomerLabelName,
+    p.additional_produktinfos,
+    p.Ingredients,
+    p.Value                                                 as productValue,
+    p.UPC,
+    p.weight,
+    c.name                                                  as country,
+    p.guaranteedaysmin,
+    p.warehouse_temperature,
+    p.description                                           as productDecription,
+    a.Name                                                  as allergen,
+    nf.Name                                                 as nutritionName,
+    pn.SeqNo                                                as NutritionSeqNo,
+    pn.nutritionqty,
+    coalesce(NULLIF(bomProduct.CustomerLabelName,''), bomProduct.Name) as componentName,
```

Comment:
```suggestion
     coalesce(NULLIF(TRIM(bomProduct.CustomerLabelName),''), bomProduct.Name) as componentName,
```

..also guard against e.g. `' '`